controllers/ArticleController.js

```diff
@@ -0,0 +1,250 @@
+   */
+  static listAllArticles(req, res) {
+    return Article
+      .findAll({
+        include: [{
+          model: User,
+          attributes: { exclude: ['id', 'email', 'hashedPassword', 'createdAt', 'updatedAt'] }
+        }],
+        attributes: { exclude: ['id', 'userId'] }
+      })
+      .then((articles) => {
+        /** check if there was no article created */
+        if (articles.length === 0) {
+          return res.status(200).json({
+            message: 'Your request was successful but no articles created',
+            articles,
+          });
+        }
+
+        return res.status(200).json({ articles, articlesCount: articles.length });
+      })
+      .catch(() => res.status(501).send('oops seems there is an error finding all articles'));
+  }
+
+  /**
+   * @function editArticle
+  * @summary: API controller to handle requests
+  * to edit an article
+  * @param {object} req: request object
+  * @param {object} res: response object
+  * @returns {object} api response: article object for
+  * successful requests, or error object for
+  * requests that fail
+  */
+  static editArticle(req, res) {
+    const { title, description, body } = req.body.article;
+    const { count } = req;
+    const { slug } = req.params;
+    return Article.update({
+      title,
+      description,
+      body,
+      updatedCount: Utilities.increaseCount(count)
+    }, {
+      where: {
+        slug,
+      },
+      returning: true,
+      plain: true
+    })
+      .then(result => res.status(200).json({
+        success: true,
+        article: result[1]
+      }))
+      .catch((err) => {
+        res.status(500).json({
+          errors: {
+            body: [
+              'sorry there was an error updating this article',
+              err
+            ]
+          }
+        });
+      });
+  }
+
+  /**
+ * @function deleteArticle
+* @summary: API controller to handle requests
+* to delete an article
+* @param {object} req: request object
+* @param {object} res: response object
+* @returns {object} api response: article object for
+* successful requests, or error object for
+* requests that fail
+*/
+  static deleteArticle(req, res) {
+    const { slug } = req.params;
+    Article.destroy({
+      where: { slug }
+    })
+      .then(() => res.status(204).json())
+      .catch(() => {
+        res.status(500).json({
+          errors: {
+            body: [
+              'sorry there was an error deleting this article',
+            ]
+          }
+        });
+      });
+  }
+
+  /**
+ * @function likeArticle
+* @summary: API controller to handle requests
+* to like an article
+* @param {object} req: request object
+* @param {object} res: response object
+* @returns {object} api response: article object for
+* successful requests, or error object for
+* requests that fail
+*/
+  static likeArticle(req, res) {
+    const { userId } = req;
+
+    const { slug } = req.params;
+    let articleId;
+    let likesCount;
```

Comment:
This can be on the same line as `let articleId`

Reply:
Alright, will fix this